lpcompiler.pas

```diff
@@ -1814,7 +1832,7 @@ function TLapeCompiler.ParseType(TypeForwards: TLapeTypeForwards; addToStackOwne
       else if (Tokenizer.Tok = tk_kw_Private) then
         BaseType := ltScriptMethod;
 
-      Expect([tk_kw_Function, tk_kw_Procedure], True, False);
+      Expect([tk_kw_Function, tk_kw_Procedure, tk_kw_Operator], True, False);
```

Comment:
Note to self: This should probably be added back, in order to support operator imports.